homeassistant/components/alarmdecoder/alarm_control_panel.py

```diff
@@ -20,6 +21,15 @@
 SERVICE_ALARM_TOGGLE_CHIME = "alarmdecoder_alarm_toggle_chime"
 ALARM_TOGGLE_CHIME_SCHEMA = vol.Schema({vol.Required(ATTR_CODE): cv.string})
 
+SERVICE_ALARM_KEYPRESS = "alarmdecoder_alarm_keypress"
```

Comment:
We don't need to prefix with the integration name when registering the service under the integration domain.